routes/api.php

```diff
@@ -21,3 +21,8 @@
 Route::get("/states", "DataControllers\StateController@getAllStates")->name("states-api");
 Route::get("/counties/{state_id?}", "DataControllers\CountyController@getCountiesInState")->name("counties-api");
 Route::get("/cities/{county_id?}", "DataControllers\CityController@getCitiesInCounty")->name("cities-api");
+
+Route::get("/countie/{state_id?}", "RegulationController@getCountiesInState")->name("my-counties-api");
+Route::get("/city/{county_id?}", "RegulationController@getCitiesInCounty")->name("my-cities-api");
```

Comment:
Could we remove these two APIs? (Since they are duplicates of lines 22-23.)

Just have your code call the "counties-api" and "cities-api" routes.